derp/derp_server.go

```diff
@@ -889,6 +981,96 @@ func (c *sclient) sendPacket(srcKey key.Public, contents []byte) (err error) {
 	return err
 }
 
+// AddPacketForwarder registers fwd as a packet forwarder for dst.
+// fwd must be comparable.
```

Comment:
What happens if it's not?

Reply:
Runtime panic when using an interface as a map key:https://play.golang.org/p/YTAmMf0PIcQ